perf/perf_rawevents.py

```diff
@@ -86,17 +93,24 @@ def error_check(self):
                 self.log.info("Failed command: %s", self.fail_cmd[cmd])
             self.fail("perf_raw_events: refer log file for failed events")
 
-    def test_raw_code(self):
-        file_name = 'raw_codes_p' + self.cpu_family
-        perf_flags = "perf stat -e r"
-        self.run_event(file_name, perf_flags)
-        self.error_check()
-
-    def test_name_event(self):
-        file_name = 'name_events_p' + self.cpu_family
-        perf_flags = "perf stat -e "
-        self.run_event(file_name, perf_flags)
-        self.error_check()
+    def test(self):
```

Comment:
Can we avoid repetitive code here? For loop can be used to make it compact.
for revision in ['004b', '004e', '0080']:
     for event in ['raw', 'name']:
             self.run_event()
             self.error_check()

few if conditions will be need to build the raw_codes_<> and raw_events_<> strings